src/brod_consumer.erl

```diff
@@ -160,6 +160,22 @@ stop(Pid) -> safe_gen_call(Pid, stop, infinity).
 %%     offset to a persistent storage, then start (or restart) with
 %%     last_processed_offset + 1 as the begin_offset to proceed.
 %%     By default, it fetches from the latest available offset.
+%%   acked_offset (optional, default = begin_offset - 1)
+%%     A subscriber may limit the prefetch window on startup with this
+%%     option. The consumer will assume offsets from acked_offset up
+%%     to but not including the effective begin_offset it calculates
+%%     will be acked in the future by the subscriber. It will reduce
+%%     the prefetch window with the number of such offsets.
+%%
+%%     This feature is most useful when the brod_consumer process
+%%     crashes, but the subscriber process survives and attempts to
+%%     subscribe again once the brod_consumer is restarted. In this
+%%     case the subscriber may still have some unacked messages
+%%     received from the crashed brod_conumser. It needs to restart
+%%     from the last offset + 1 it has seen, so it won't receive
```

Comment:
"from the last offset + 1 it has seen" is ambiguous to me; does it mean "from one past the last offset it has seen" or does it mean (constraint-programming style) "from Y, where Y+1 is the last offset it has seen"?